twistedchecker/checkers/python3.py

```diff
@@ -1,187 +0,0 @@
-"""
-Checker for python3 compatibility issues.
```

Comment:
There are projects other than Twisted which use twistedchecker (axiom and vertex come to mind) which are not finished porting to python 3, so this seems premature.